dummyserver/app.py

```diff
@@ -353,29 +371,69 @@ async def pyodide_upload() -> ResponseReturnValue:
     return await make_response("Uploaded file correct", 200)
 
 
+def _find_built_wheel() -> Path | None:
+    wheel_folder = Path(__file__).parent.parent / "dist"
+    wheels = list(wheel_folder.glob("*.whl"))
+    wheels = sorted(wheels, key=lambda w: w.stat().st_mtime)
+    if len(wheels) > 0:
+        return wheels[-1]  # newest wheel
+    else:
+        return None
+
+
+def _rewrite_pyodide_lock() -> bytes | None:
```

Comment:
What is rewriting the lockfile needed for? In `emscripten/conftest.py`, you run `loadPackage` the `urlib3.whl` directly, which I'd think would be good enough to make sure you're getting the right version. Is there some place that installs it by name?